js/index.js

```diff
@@ -0,0 +1,33 @@
+/*!
+ * Copyright (c) 2023 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const context = require('./context');
+
+const constants = {
+  // default context output filename under ./contexts/
+  CONTEXT_FILENAME: 'vdl-aamva-v1.jsonld',
+  // default context well known URL id
+  CONTEXT_URL: 'https://w3id.org/vdl/aamva/v1',
+  // default context CBOR-LD codec id
+  // value between 0x0 and 0x7FFF for globally registered term codec values
+  // or >= 0x8000 for app-specific local terms
+  CBORLD_VALUE: 0x0
+};
+
+// map of all context URLs to context data
+const contexts = new Map();
+contexts.set(constants.CONTEXT_URL, context);
+
+// map of all context URLs to CBOR-LD codec ids
+const appContextMap = new Map();
+appContextMap.set(constants.CONTEXT_URL, constants.CBORLD_VALUE);
+
+module.exports = {
+  CONTEXT: context,
+  CONTEXT_URL: constants.CONTEXT_URL,
+  appContextMap,
+  constants,
+  contexts
```

Comment:
@davidlehn For some reason, these three variables are not getting able to be imported from this library. Here is the kind of error that we get:
```
SyntaxError: Named export 'contexts' not found. The requested module '@digitalbazaar/vdl-aamva-context' is a CommonJS module, which may not support all module.exports as named exports.
```
The only thing that confuses me here is that this error is _only_ visible with these variables but not the other two. We really only need the other two for our (and most) use cases anyway, but I wonder if you can see why there was an issue with these variables and if it's worthwhile to fix it.

Reply:
The CJS/ESM story here is a bit convoluted due to history of these context modules.  I may have fixed it with some more explicit ESM bits.  But also see: https://github.com/digitalbazaar/vdl-aamva-context/pull/2 that flips this to a module that generates CommonJS.